webapp/packages/plugin-data-viewer-result-set-grouping/src/DVGroupingColumnEditorDialog/GroupingTableItem.m.css

```diff
@@ -0,0 +1,3 @@
+.tableColumnValue {
+  width: 100%;
+}
```

Comment:
what it's for?

Reply:
<img width="676" alt="image" src="https://github.com/dbeaver/cloudbeaver/assets/75555523/1bd2ba86-50b7-430c-87cd-14c9a4bb182b">

Reply:
But now I see that TableColumnValue has prop width, so will remove that style.